test/chpldoc/nodoc/sub_test

```diff
@@ -0,0 +1,6 @@
+$CHPL_HOME/util/test/sub_test $1
+touch NOEXEC
+$CHPL_HOME/util/test/sub_test $1doc
+rm NOEXEC
+
+exit $?
```

Comment:
I think the newer format for the sub_test files (e.g. https://github.com/chapel-lang/chapel/blob/master/test/chpldoc/sub_test ) is needed here.